tests/unittests/OperatorTest.cpp

```diff
@@ -64,6 +64,77 @@ static Tensor createTensorConditionallyQuantized(ElemKind T,
   return isQuantizedElemKind(T) ? Tensor(T, dims, 1.0, 0) : Tensor(T, dims);
 }
 
+// Helper to test SpaceToDepth using \p DTy.
+template <typename DataType>
+static void testSpaceToDepth(glow::PlaceholderBindings &bindings,
+                             glow::Module &mod, glow::Function *F,
+                             glow::ExecutionEngine &EE, ElemKind DTy) {
+  auto *in = createPlaceholderConditionallyQuantized(mod, DTy, {2, 2, 4, 4},
+                                                     "in", false);
+  auto *tri = F->createTranspose("sptdTransposeIn", in, {0, 2, 3, 1});
+  auto *stdn = F->createSpaceToDepth("spacetodepth", tri, 2);
+  auto *tro = F->createTranspose("sptdTransposeOut", stdn, {0, 3, 1, 2});
+  auto *save = F->createSave("save", tro);
+  auto *result = bindings.allocate(save->getPlaceholder());
+
+  /*
+    Example for first batch.
+  FROM:
+  [0  1  2  3]    [ 0  -1  -2  -3]
+  [4  5  6  7]    [-4  -5  -6  -7]
+  [8  9  10 11]   [-8  -9  -10 -11]
+  [12 13 14 15]   [-12 -13 -14 -15]
+
+  TO:
+  [0,  2] [ 0,  -2] [1, 3]  [-1, -3]  [4,  6]  [-4,  -6]  [5, 7]   [-5,  -7]
+  [8,  10] [-8, -10] [9, 11] [-9, -11] [12, 14] [-12, -14] [13, 15] [-13, -15]
+  */
+
+  bindings.allocate(in)->getHandle<DataType>() = {
+      0, 1,   2,   3,   4,  5,  6,   7,   8,  9,  10,  11,  12,  13,  14,  15,
```

Comment:
Can we please reformat as this:
```
c=0:
[0, 2]
[8, 10]
c=1:
[0, -2]
[-8, -10]
c=2:
...
```
And input too, please.
Otherwise it was difficult for me to figure out what this op does, and how to read it.

Reply:
OK, modified comment.